crates/kitsune_p2p/dht/src/spacetime/segment.rs

```diff
@@ -231,7 +243,8 @@ pub(super) fn bounds<N: From<u32>>(
     (start.into(), start.wrapping_add(len).wrapping_sub(1).into())
 }
 
-pub(super) fn bounds64<N: From<i64>>(
+// TODO is wrapping meaningful for time? Should we saturate instead?
```

Comment:
Yes, we expect this to never overflow, so saturation makes more sense

Reply:
Cool, I'll leave this as a TODO for now then